src/phylum/ci/ci_base.py

```diff
@@ -512,15 +519,15 @@ def analyze(self) -> None:
 
         if self.all_deps:
             LOG.info("Considering all current dependencies ...")
-            base_pkgs = []
-            total_packages = {pkg for depfile in self.depfiles for pkg in depfile.current_deps}
-            LOG.debug("%s unique current dependencies", len(total_packages))
+            base_packages = []
+            current_packages = {pkg for depfile in self.depfiles for pkg in depfile.deps}
+            LOG.debug("%s unique current dependencies from %s file(s)", len(current_packages), len(self.depfiles))
```

Comment:
I'm not sure why this log is unique to `--all-deps`. I think it would make sense to either move it out of the `if` statement or delete it (my preference is always to delete it).

Reply:
It has been moved out, to be displayed regardless of whether `--all-deps` is supplied.